drivers/classic/src/write_header.c

```diff
@@ -106,7 +106,7 @@ write_header(out_data_file_struct *out_data_files,
             // dt
             Nbytes1 += sizeof(int);
 
-            // start date (year, month, day, hour)
+            // start date (year, month, day, sec)
```

Comment:
and line 81 should replace "int" with "double" (although it's just a comment statement)